Wendy/assignment1Answers.html

```diff
@@ -0,0 +1,219 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>JavaScript Immersion Assignment 1 || Wendy Goodwin</title>
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+	    <link href="css/main.css" rel="stylesheet">    
+	    <link href="css/mystyles.css" rel="stylesheet">
+	</head>
+
+	<body>
+
+		<div id="wrap">
+			<div class="title">
+				<h1> JavaScript Immersion Assignment I </h1>
+				<h3> by Wendy Goodwin </h3>	
+				<h4> 
+
+					<a href="https//github/portlandcodeschool/jse=fall15-1">	View assignment specifications</a>
+
+
+				</h4>
+			</div>
+			<div class="row" id="part1">
+				<h3> Part I </h3>
+				<div class="col-md-4">
+					<h3>a) "1" == 1  true.</h3>
+					<p>The == equality operator says "kind of the same" and they are.  Javascript reads the equality operator from right to left and changes the operators to the same type if needed.  
+
+					<h3>b) "1" === 1 false.</h3>
+					<p>The "===" is "exactly equal to"  No room for kind of.</p>
+
+					<h3>c) x == 'x'  false.</h3>
+					<p>  If x is defined as a string, the 'x' is saying string 'x' and if your string is for example var x = 'snowman', 'snowman' != 'x'.  If x is a number, it is still false because a number is not a string.</p>
+
+
+					<h3>d) x == (x+'') false.</h3>
+					<p>This is false because the '' empty quotes make things falsy!</p>
```

Comment:
This will actually return true except for a few specific cases (further explained in the homework solutions). x is evaluated against x, more or less ignoring the empty string.